README.md

```diff
@@ -116,6 +116,18 @@ A more detailed description of the ways to filter resources:
    
    The tag filter can be negated by surrounding the regex with `NOT(...)`
 
+   Resources not matching tag can be removed with Untagged: true
+
+    aws_instance:
+      - tags:
+          Persist: NOT(true)
```

Comment:
>  In the above example all aws instances without the `Persist: true` tag ... will be deleted.

Shouldn't this be "all AWS instances which tags do not match "Persist: NOT(true)" instead? Please correct me if I am wrong.